src/make.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function make() {
+    let r = Array.from(arguments);
+  
+    const func = function ret(...args) {
+      if (typeof(args[0]) !== "function") {
+        r.push(...args);
+        return ret;
+      } else {
+        return r.reduce((prev, current) => args[0](prev, current));
+      }
+    };
+  
+    return func; 
+  }
+  
```

Comment:
Incorrect indents: somewhere there are 2 spaces, somewhere there are 4 spaces
the project indents should be the same

In the whole project indents should be the same